cmd/noisd/root.go

```diff
@@ -93,14 +93,18 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 }
 
 func initRootCmd(rootCmd *cobra.Command, encodingConfig params.EncodingConfig) {
+	debugCmd := debug.Cmd()
+	debugCmd.AddCommand(Bech32EncodeCmd())
+	debugCmd.AddCommand(Bech32DecodeCmd())
```

Comment:
Is this some standard secition of Cosmos SDK? Could we also make a bech32 section? Like `noisd bech32 encode`

Reply:
yeah let me move it to a bech32

Reply:
moved to its own `noisd bech32 encode/decode`